docker-images/syntax-highlighter/crates/scip-syntax/src/bin/scip-perf.rs

```diff
@@ -58,11 +91,16 @@ fn measure_parsing() {
 
     let root = Path::new(&args.root_dir);
 
-    let mut timings = parse_files(root);
+    let mut timings = parse_files(root, args.language);
     timings.sort_by(|a, b| a.duration.cmp(&b.duration));
     println!("Slowest files:");
     for timing in timings.iter().rev().take(10) {
-        println!("{}: {:?}", timing.filepath, timing.duration);
+        println!(
+            "{} ({}kb): {:?} ",
```

Comment:
```suggestion
            "{} ({}kb): {:?}",
```

nit: extraneous space at end before newline